### PR TITLE
Rename `six` project to `rtloader`.  

### PR DESCRIPTION
**Pull request by derekwbrown**
Phase 1 of two; still need to rename internal libraries/variables/headers.

### What does this PR do?

Renames the build tooling for the `six` project to `rtloader`.  Six was ambiguos;
and the copy of `datadog-agent-six` exceeded the max file path on windows.

### CODE REVIEW

**Review by masci — DISMISSED** [review]
This is a waste of time IMO. This PR alone is huge and it's going to leave an inconsistent state when merged. The problem of `datadog-agent-six` being too long on Windows should be fixed elsewhere.

File: tasks/rtloader.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def build(ctx, install_prefix=None, cmake_options=''):
     here = os.path.abspath(os.path.dirname(__file__))
     dev_path = os.path.join(here, '..', 'dev')
 
+    print("cmake options {}".format(cmake_options))
```
Comment (hush-hush) [inline tasks/rtloader.py]: we should remove this since the `-e` option of invoke already print the whole comand

**Review by hush-hush — APPROVED** [review]
nit-pick but look good overall

**Comment by albertvaka** [comment]
I think this can be closed @derekwbrown ?

**Pull request closed without merging**